### ISSUE TITLE
Transformations are blurry and reconstructions good

### ISSUE THREAD

**Issue by kpagels** [junyanz/pytorch-CycleGAN-and-pix2pix#1012]
Hi, I am currently using your CycleGAN code. I am however experience that the transformations are blurry and totally gray, where the reconstructions are really nice. I tried to reduce the learning rate from 0.0002 to 0.00002, but that didn't really work. Do you know if anything else would work? I find it strange that the reconstructions are great, but the transformations are really bad. 

Do you have any suggestions for this matter?

**Comment by junyanz** [comment]
It happens when the transformation is difficult to learn or two domains look drastically different. Not sure if reducing the learning rate will help in your case. I will try (1) adding some paired data or (2) using small cropped patches rather than the entire image.

**Comment by wave-transmitter** [comment]
I opened a new issue  here #1286  since I noticed a similar to some extent behavior of CycleGAN in case of RGB to IR/thermal translation. @kpagels if you have any thoughts please feel free to share.

**Comment by junyanz** [comment]
@duke023456  It is hard for CycleGAN to preserve the text in an image. If you can detect the text using an OCR method, you can try to preserve them using an additional loss.

**Comment by junyanz** [comment]
You can detect the region that contains the text using an OCR detection method, and add a loss function to preserve the pixels within the region.
